salt/tests/unit/modules/test_cri.py

```diff
@@ -0,0 +1,194 @@
+import json
+
+from parameterized import parameterized
+
+from salttesting.mixins import LoaderModuleMockMixin
+from salttesting.unit import TestCase
+from salttesting.mock import MagicMock, patch
+
+from tests.unit import utils
+
+import cri
+
+
+IMAGES_LIST = [{
```

Comment:
Since we are starting to have variables of 21+ lines, what is the good practice here...
 - To define a global helper file that contains these variables and simply import them from there or
 - Move the variable declaration to the bottom of this file

Reply:
To me I think it's good to have example of output you get from command in test so that it's easier to understand the code, anyway to me here it's still simple ones so it's ok to have them directly in the python test file IMHO.

For complicated ones I move it to file check https://github.com/scality/metalk8s/pull/2656